TempRes3.m

```diff
@@ -1,22 +1,24 @@
 % The variable freqCol (below) is the column containing the frequency values in the
 % csv file.
-freqCol = 2;
+freqCol = 1; %should be 1 for pulse trains and 2 for whistles from ROCCA
```

Comment:
Perhaps this can be amended into an if statement which can be a button the user presses (depending on whether they are analysing pulse trains or whistles)